DigitalLearningSolutions.Web/Controllers/TrackingSystem/CentreConfiguration/CentreConfigurationController.cs

```diff
@@ -64,7 +74,7 @@ public IActionResult EditCentreWebsiteDetails()
         {

```

Comment:
Could we write some controller tests for this where we mock the maps api helper? 🙏